tests/utils.py

```diff
@@ -87,3 +87,21 @@ def check_server_ready(protocol="grpc"):
         output = buf.getvalue()
     output = json.loads(output)
     return output["ready"]
+
+
+# Context Manager to start and kill a mock server running in background and used by testing functions
+class MockServer:
+    def __init__(self, repo=None, mode="local", timeout=60):
+        self.repo = repo
+        self.mode = mode
+        self.timeout = timeout
+
+    def __enter__(self):
+        run(["remove", "-m", "all"])
```

Comment:
Can you make the following tweaks for use of `clear()`?
1. move the `_clear()` helper into `utils.py` and use it here?
2. Add a `if repo:` clause to the `_clear()` helper, I just realized it's missing. Will look like [this](https://github.com/triton-inference-server/triton_cli/blob/b1f7c40f951ebe167aa825abe8b5804fd4606785/tests/test_cli.py#L55-L56).
3. Update the other calls to do `utils.clear()` instead of `self._clear()`

Reply:
Would it make sense to move all the helper functions into `utils.py`? It will reduce the replication of helper functions between `test_cli.py` and `test_e2e.py`.

Reply:
Sounds good if being replicated